intro-to-rails-2-relationships/ar-validations.md

```diff
@@ -1,10 +1,14 @@
 # Active Record Validations
 
+<iframe src="https://adaacademy.hosted.panopto.com/Panopto/Pages/Embed.aspx?pid=331d23d5-762b-4f2c-9554-ac5e0133a56d&autoplay=false&offerviewer=true&showtitle=true&showbrand=false&start=0&interactivity=all" height="405" width="720" style="border: 1px solid #464646;" allowfullscreen allow="autoplay"></iframe>
+
 ## Learning Goals
 
+By the end of this lesson we will be able to:
+
 - Explore the _validations library_ provided by Active Record
 - Discuss the role _validations_ play in data management
-- Guide users towards providing better data
+- Guide users towards providing better data with validations and feedback messages
```

Comment:
```suggestion
- Guide users towards providing better data by using validations and feedback messages
```

Reply:
This just feels more clear to me b/c otherwise it reads like the user is providing data with validations and feedback in the data